addons/frag/functions/fnc_dev_fragCalcDump.sqf

```diff
@@ -0,0 +1,76 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger, based on fnc_dev_debugAmmo by "ACE-Team"
+ * Dumps all ammo types to see if there's any reason to spawn fragments
+ * given power, distance, and lifetime of each fragement. Good for grasping
+ * the values used in shouldFrag to cull non-fragementing rounds
+ *
+ * Arguments:
+ * 0: _dispAll <BOOL> - Display rounds that will never frag (power < 5).
+ *           Default value false
+ * 1: _minFrgPowRng <FLOAT> - minimum range for sqrt power calculation
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [false, 10] call ace_frag_fnc_fragoCalcDump
+ *
+ * Public: No
+ */
+
+params [["_dispAll", false, [false]], ["_minFrgPowRng", 35, [123]]];
+
+#define DT 0.01
+
+private _allAmmoConfigs = configProperties [configFile >> "cfgAmmo", "isClass _x && !('ace_frag' in configName _x)", true];
+private _processedCfgAmmos = [];
+
+
+
+private _nPrinted = 0;
+
+diag_log text "//****************** fragCalcDump Beg ******************//";
+// Processing ammo types
+{
+    private _ammo = toLower configName _x;
```

Comment:
```suggestion
    private _ammo = toLowerANSI configName _x;
```

Reply:
Addressed in https://github.com/acemod/ACE3/pull/9728/commits/3f45f7735b6b6c50f01d6fcb50db08f20548c0c4